templates/default/condaexport.sh.erb

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# conda list [-c $1]
+
+channel=
+if [[ $# -ne 5 ]]; then
+  echo "incorrect usage"
+  echo "exportdir project host environment user"
+  exit 1
+fi
+
+CONDA_USER=<%= node['conda']['user'] %>
+GLASSFISH_USER=<%= node['glassfish']['user'] %>
+
+EXPORTDIR=$1
```

Comment:
You can check to make sure this is a valid directory. 
See example code in jupyter.sh.erb - line 72.

    if [ ! -d "$2" ] ; then
	echo "Invalid PRIVATE_DIR directory: $2"
	exit 11
    fi